service-runner.js

```diff
@@ -153,11 +175,12 @@ ServiceRunner.prototype._runMaster = function() {
         if (!self._shuttingDown && !self._inRollingRestart) {
             var exitCode = worker.process.exitCode;
             self._logger.log('error/service-runner/master',
-                    'worker' + worker.process.pid
-                    + 'died (' + exitCode + '), restarting.');
+                    'worker ' + worker.process.pid
```

Comment:
I'm sooo happy you noticed the missing spaces surrounding the PID. They were driving me nuts a bit :))))